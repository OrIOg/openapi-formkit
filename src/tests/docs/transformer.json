{
  "openapi": "3.0.0",
  "info": {
    "title": "Simple API overview",
    "version": "1.0"
  },
  "paths": {
    "/transformers-examples": {
      "get": {
        "operationId": "listExamples",
        "summary": "List demo examples",
        "parameters": [
          {
            "in": "query",
            "name": "number-minmax",
            "required": true,
            "schema": {
              "type": "number",
              "minimum": 17,
              "maximum": 34
            }
          }
        ],
        "responses": {
          "200": {
            "description": "200 response",
            "content": {
              "application/json": {
                "schema": {
                  "type": "array",
                  "items": {
                    "type": "string"
                  }
                }
              }
            }
          }
        }
      }
    }
  },
  "components": {
    "schemas": {
      "Example": {
        "description": "Example obj",
        "type": "object",
        "properties": {
          "name": { "type": "string" },
          "description": { "type": "string" }
        },
        "required": ["name", "description"]
      }
    }
  }
}
