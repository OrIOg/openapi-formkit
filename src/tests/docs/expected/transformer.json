{
    "/transformers-examples": {
      "get": {
        "$cmp": "FormKit",
        "props": { "type": "form", "method": "get" },
        "children": [
          {
            "$cmp": "FormKit",
            "props": {
              "type": "range",
              "name": "number-minmax",
              "label": "number-minmax",
              "step": 0.1,
              "min": 17,
              "max": 34,
              "validation": [["(200)required"]]
            }
          }
        ]
      }
    }
  }
  